packages/react/src/transactions/adapters/publication-call-gateways/__tests__/PostCallGateway.spec.ts

```diff
@@ -71,7 +71,7 @@ const mandatoryFallbackMetadata = (request: CreatePostRequest) => ({
   mainContentFocus: PublicationMainFocus[request.contentFocus],
 });
 
-describe(`Given an instance of ${PostCallGateway.name}`, () => {
+describe(`Given an instance of ${CreatePostCallGateway.name}`, () => {
```

Comment:
Should we rename the test files for Post and Comment call gw as well?

Reply:
oh, good catch. Fixed